src/registrar/templates/admin/analytics.html

```diff
@@ -27,28 +27,35 @@ <h2>At a glance</h2>
       <div class="module height-full">
         <h2>Current domains</h2>
         <div class="padding-top-2 padding-x-2">
-          <ul class="usa-button-group">
+          <ul class="usa-button-group analytics-button-group">
```

Comment:
If this is needed, suggest using a CSS descriptive class such as min-with-160 instead of analytics-button-group